google/cloud/bigtable/table.h

```diff
@@ -831,12 +829,8 @@ class Table {
    * @par Example
    * @snippet data_async_snippets.cc async read rows
    */
-  template <typename RowFunctor, typename FinishFunctor>
```

Comment:
I did not think about this before: in #9272 we imposed an additional requirement: anything that goes into a `std::function<>` must be copy-constructible, but `AsyncRowReader` required (only?) move-constructible.

Thougths?